src/class/eventManagement/incomer.class.ts

```diff
@@ -128,7 +128,8 @@ export class Incomer <
     });
 
     if (
-      (this.prefix === "test" || this.prefix === "development") && !options.externalsInitialized
+      (this.prefix === "test" || this.prefix === "development") &&
+      (typeof options.externalsInitialized !== "undefined" && !options.externalsInitialized)
```

Comment:
Et ça marcherai pas en faisant ça
```suggestion
     options.externalsInitialized === false
```